.github/workflows/nightly-scans.yaml

```diff
@@ -193,20 +229,58 @@ jobs:
           ]
     steps:
       - uses: actions/checkout@v3
+      - name: Login with ghcr
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: Install yq
         run: |
           sudo snap install yq
       - uses: ./.github/actions/k8s-version-config
         name: Setup k8s cluster
         with:
           k8s-version: ${{ matrix.k8s-version }}
-          load-local-image: false
       - name: Run pre-config and workload integration tests
         run: |
-          bash tests/integration/integration-test.sh "nightly-pre-and-workload"
+          bash tests/integration/integration-test.sh "pre-and-workload"
         shell: bash
       - name: Display k8s logs if integration test failed
         if: failure()
         run: |
           kubectl logs -n connaisseur -lapp.kubernetes.io/name=connaisseur --prefix=true
         shell: bash
+  
+  cleanup-registry:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cleanup test images in 'connaisseur-test'
+        uses: snok/container-retention-policy@v1
+        with:
+          image-names: connaisseur-test
+          cut-off: three weeks ago UTC+1
+          timestamp-to-use: updated_at
+          account-type: org
+          org-name: sse-secure-systems
+          token: ${{ secrets.GHCR_PAT }}
```

Comment:
I guess that's a personal token of yours which is only allowed to read/write the registry?

Reply:
correct, actually only `packages:delete` which includes `read`